src/main/java/com/spotify/confidence/GrpcEventUploader.java

```diff
@@ -23,6 +28,17 @@ class GrpcEventUploader implements EventUploader {
     this.managedChannel = managedChannel;
     this.stub = EventsServiceGrpc.newFutureStub(managedChannel);
     this.clock = clock;
+    try {
+      final Properties prop = new Properties();
+      prop.load(this.getClass().getResourceAsStream("/version.properties"));
```

Comment:
we are using the same block to get the version of the sdk in [here](https://github.com/spotify/confidence-openfeature-provider-java/blob/8271811da17a59d30539900102ba78bf9f1bff2d/src/main/java/com/spotify/confidence/GrpcFlagResolver.java#L29), do you think if it makes sense to extract this part to a utility function giving back the sdk version?